packages/appeal-reply-service-api/api/openapi.yaml

```diff
@@ -0,0 +1,355 @@
+openapi: 3.0.3
+info:
+  title: Appeals Service API Documentation
```

Comment:
Use correct name - I suspect this will need updating throughout as it's likely to be using old spec

Reply:
Done. Also deleted some code that's currently not relevant (structure of appeal).